packages/mongo-nav/src/MongoNav.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import OrgNav from './org-nav/index';
+import ProjNav from './proj-nav/index';
+import { DataInterface, Product, OverridesInterface, NavItem } from './types';
+interface MongoNavInterface {
+  activeProduct: Product;
+  activeNav?: NavItem;
+  data: DataInterface;
+  onOrganizationChange?: React.ChangeEventHandler;
+  onProjectChange?: React.ChangeEventHandler;
+  constructOrganizationURL?: (orgID: string) => string;
+  constructProjectURL?: (orgID: string, projID: string) => string;
+  overrides?: OverridesInterface;
+  showProjNav?: boolean;
+}
+
+export default function MongoNav({
+  activeProduct,
+  activeNav,
+  data,
+  overrides,
```

Comment:
What if we un-nest this by one level? Such that we have a `urls` and a `hosts` props directly on the component? Not sure we get much benefit nesting them within another object.

Reply:
Expanding on that...the biggest source of complexity for all these components is understanding what the default URL is and when to use a URL that is explicitly being set.

I think what may make that easier to reason about is if, within this component, `MongoNav`, we do all the logic around urls and default urls here.

Such that what we produce from that is the final shape of the URLs. So that we can just pass a `urls` prop to `OrgNav` and `ProjectNav` and they just use those values directly.

So all that logic is consolidated in one location, making it easy to reason about how some value ended up somewhere else.

Reply:
Done